README.md

```diff
@@ -412,6 +412,12 @@ The `mst-gql` command currently accepts the following arguments:
   - a graphql files, like `schema.graphql`
   - a parsed graphql file, like `schema.json`
 
+### Config
+
+`mst-gql` also supports [cosmiconfig](https://github.com/davidtheclark/cosmiconfig) as an alternative to using cli arguments.
+
+and example config can be found in example 2 [here](https://github.com/mobxjs/mst-gql/blob/master/examples/2-scaffolding/mst-gql.config.js).
```

Comment:
A sample config can be found in [Example 2](https://github.com/mobxjs/mst-gql/blob/master/examples/2-scaffolding/mst-gql.config.js).